### DISCUSSION TITLE
What's wrong with Github Copilot?

### DISCUSSION THREAD

**Discussion by ANLINGG** [community/community#112097] [Copilot Conversations]
### Select Topic Area

Question

### Body

I have successfully applied for the student pack and passed the verification and used it for a while, why did it suddenly tell me today that I don’t have the access to use GitHub copilot? I was coding just now, and then copilot shut down all of  a sudden.

**Comment by hamd34** [comment]
Hi!
If you suddenly lost access to GitHub Copilot after using it through the Student Developer Pack, consider checking your student verification status, confirming the pack's expiration date, ensuring compliance with GitHub's terms of service, and addressing any potential technical issues. If the problem persists, contact GitHub support for assistance.

**Reply by ANLINGG** [reply]
It's kinda weird i have to say...After I logged in to Education GitHub, no matter how I modified the link, I could only go to the GitHub global campus page, which prevented me from checking if my verification status was normal. Also, when I checked GitHub status, I saw that copilot status was degraded, and I don’t know what happened out there.

**Reply by ANLINGG** [reply]
the problem seem to get back to normal somehow...thanks anyway!
